### PR TITLE
glob support in manifestPath

### PR DESCRIPTION
**Pull request by bushee**

### CODE REVIEW

**Comment by bushee** [comment]
@reagent sorry to bother, you are the first to pop up on mention list, and @greypants seems to be out of this project. Could you please consider my PR? :)

**Comment by reagent** [comment]
hey @bushee -- I don't think we maintain this anymore, but I just gave you push access to the repo if you want to merge your changes.

**Comment by bushee** [comment]
@reagent Thanks. Could you please add me as a collaborator to https://www.npmjs.com/package/rev-replace-loader as well, so I can publish it afterwards?

**Comment by reagent** [comment]
I may have jumped the gun on this -- there wasn't consensus on our side about how we're handling this project and how it affects other projects on our github that might depend on it.  I chatted with @nhunzaker and he's going to check to see what we need to do to get this released without causing potential issues.

**Comment by greypants** [comment]
Feel free!

File: index.js [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
 }
 
 function requireManifest(manifestPath) {
-  try {
-    return require(manifestPath)
-  } catch(error) {
-    return false
-  }
+    try {
+        return require(process.cwd() + path.sep + manifestPath);
```
Comment (greypants) [inline index.js]: I might use `path.resolve()` here, instead of string concat and path.sep.

Comment (nhunzaker) [reply index.js]: 👍, good catch. I realize I haven't even looked at this PR 🙈

**Comment by nhunzaker** [comment]
Hey @bushee, I apologize for the delay. So here's where we're at. 

We do not actively use this project at Viget, and would like to find a new maintainer for it. That could be you. If so, we'd do the following:

1. We'd transfer ownership of this repo to you
2. We'd grant you access to the project on NPM
3. You'd be free to make changes as you see fit

Right now we're looking into all of the projects that use `rev-replace-loader` to make sure their package.json entries reference hard versions, or use lockfiles. 

I think the best course of action would be to send out a major release, giving you some padding if any projects accidentally pull in a newer version with breaking changes. This probably isn't the case, but we're checking that now.

In the mean time, I'll review this PR and focus on getting it merged and shipped.

File: index.js [hunk 2]
```diff
@@ -1,26 +1,36 @@
...
+    } catch (error) {
+        return false;
+    }
 }
```
Comment (nhunzaker) [inline index.js]: Did this go from 2 to 4 spaces? Do you mind reformatting it back to 2 spaces?

File: index.js (hunk 1 above)
Comment (nhunzaker) [reply index.js]: Fun fact though: I didn't know `path.sep` was a thing.

Comment (bushee) [reply index.js]: @greypants funny thing is I usually use `path.resolve()`, hence I have no idea what happened in my brain here ;)

File: index.js (hunk 2 above)
Comment (bushee) [reply index.js]: I'm used to 4 spaces indentation, but you're right - I shouldn't have changed such a thing in one's project.

As of version, this PR is not a breaking change, so I don't think major version should be bumped, but it's up to you.

Comment (bushee) [reply index.js]: I might be the one to have this project transferred to, no problem here :)

**Comment by nhunzaker** [comment]
Sorry for the delay. This looks good. I'll verify it and deploy it. Then I'll transfer ownership of the repo over to you.

**Comment by nhunzaker** [comment]
@bushee same name on NPM?

**Comment by nhunzaker** [comment]
@bushee Sorry, final thing! Looks like I can't transfer the repo to you because you already have a fork. Would you be willing to remove your fork?

**Comment by bushee** [comment]
@nhunzaker yup, https://www.npmjs.com/~bushee is me.
Now that you've merged this PR, I deleted my fork. Feel free to transfer ownerships to me, thanks :)

**Comment by nhunzaker** [comment]
Done! You should receive an email shortly.

**Comment by bushee** [comment]
Received and accepted. Thanks!

**Pull request merged**